### DISCUSSION TITLE
Access violation error while running wireless simulation in omnet++ 6

### DISCUSSION THREAD

**Discussion by marshalbabu** [omnetpp/omnetpp#1126] [Q&A]
I have created LTE network(ned file and ini file) using simu 5G. Everything was fine but when I run simulation error shown as below.Please help.

Exception code=0xc0000005 flags=0x0 at 0x00007FFACAC121DB. Access violation - attempting to read data at address 0x0000000000000000

Simulation terminated with exit code: c0000005 Working directory: C:/omnetpp-6.0-windows-x86_64/omnetpp-6.0/myproject/first Command line: first.exe -m -n .;../simu5g/emulation;../simu5g/simulations;../simu5g/src;../inet4.5/examples;../inet4.5/showcases;../inet4.5/src;../inet4.5/tests/validation;../inet4.5/tests/networks;../inet4.5/tutorials -x simu5g.simulations.LTE.cars;simu5g.simulations.NR.cars;simu5g.nodes.cars;inet.common.selfdoc;inet.linklayer.configurator.gatescheduling.z3;inet.emulation;inet.showcases.visualizer.osg;inet.examples.emulation;inet.showcases.emulation;inet.transportlayer.tcp_lwip;inet.applications.voipstream;inet.visualizer.osg;inet.examples.voipstream --image-path=../simu5g/images;../inet4.5/images -l ../simu5g/src/simu5g -l ../inet4.5/src/INET lte.ini

Environment variables: PATH=;C:/omnetpp-6.0-windows-x86_64/omnetpp-6.0/myproject/simu5g/src;C:/omnetpp-6.0-windows-x86_64/omnetpp-6.0/myproject/inet4.5/src;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\bin;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\opt\mingw64\bin;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\usr\bin;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\mingw64\bin;;C:/omnetpp-6.0-windows-x86_64/omnetpp-6.0/ide//plugins/org.eclipse.justj.openjdk.hotspot.jre.full.stripped.win32.x86_64_17.0.1.v20211116-1657/jre/bin/server;C:/omnetpp-6.0-windows-x86_64/omnetpp-6.0/ide//plugins/org.eclipse.justj.openjdk.hotspot.jre.full.stripped.win32.x86_64_17.0.1.v20211116-1657/jre/bin;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\bin;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\opt\mingw64\bin;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\mingw64\bin;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\usr\local\bin;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\usr\bin;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\usr\bin;C:\Windows\System32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\usr\bin\site_perl;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\usr\bin\vendor_perl;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\tools\win32.x86_64\usr\bin\core_perl;C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0; OMNETPP_RELEASE=omnetpp-6.0 OMNETPP_IMAGE_PATH=C:\omnetpp-6.0-windows-x86_64\omnetpp-6.0\images

**Comment by rhornig** [comment]
This is an issue to be reported to the Simu5G repository, but be sure to provide step by steo instructions how to reproduce it. The "I run a 5G simulation as it crashed" will not be enough.

But generally it seems that you have your own C++ classes implemented so the crash is most likely in your code. You should debug it.
